### PR TITLE
borderCorner of the footer icons R now rounded🤩🤩

### LINKED ISSUE #319: Enhacement in Footer Icons
Deliverable:

- [ ] Please make the border around the icons curved around the corners as it's looking like a glitch this way.

Screenshot from 2021-01-31 14-43-47

### PR DESCRIPTION
**Pull request by ashuvssut**
## Issue that this pull request solves

 Closes #319 

## Types of changes

_Put an `x` in the boxes that apply_
- [x] New feature (non-breaking change which adds functionality)expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Other information

I have written customizable css that only require you to change 2 variables and you do not need to bother about alignment or anything to change its size. those two css variables are

```
:root{
    --r: 12px; /* corner radius */
    --w: 4px; /* width of the border */
}
```

## Screenshots
Desktop Recording 4

### CODE REVIEW

**Comment by ashuvssut** [comment]
initially I was hopeless staring at my screen for hours thinking of some approach
However I managed to do it

@Kajol-Kumari , I had a great time learning!😁😁😁thx

File: frontend/src/components/footer/footer.css [hunk 1]
```diff
@@ -177,75 +181,114 @@
...
 }
-
+.col .social li a .outer {
+  height: calc(50px + 2 * var(--w));
+  width: calc(50px + 2 * var(--w));
+  overflow: hidden;
+  border-radius: 10px;
+  position: relative;
+}
+.col .social li a .inner {
```
Comment (Kajol-Kumari) [inline frontend/src/components/footer/footer.css:191]: please put one line space here

File: frontend/src/components/footer/footer.css [hunk 2]
```diff
@@ -177,75 +181,114 @@
...
+  z-index: 1;
+  width: calc(50px + var(--w));
+  height: calc(50px + var(--w));
+  border-radius: 10px;
+  position: absolute;
+  background: #121f3a;
+  top: calc(var(--w) / 2);
+  left: calc(var(--w) / 2);
+}
+a > span {
```
Comment (Kajol-Kumari) [inline frontend/src/components/footer/footer.css:201]: same here

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @ashuvssut the feature is working perfectly fine and I am glad that you got to learn something here :)

Just format the code by using preetier and leave one line space btw each class in the css file and it will be good to go :smile:

**Comment by ashuvssut** [comment]
@Kajol-Kumari i my pushing changes in 5 mins

**Comment by ashuvssut** [comment]
@Kajol-Kumari , in my previous DWoC project doc2pen, whenever I pushed such new minor changes like this, that repo's maintainer asked me to do a squash commit. I know that maintainers can choose to do a **squash commit n merge** themselves.
Do you have any idea why he always said to do a squash commit to me instead of doing it hemself?
I never asked him this question coz he might get offended or something😗🤔

**Comment by ashuvssut** [comment]
Mam please put labels for DWoC and SWoC on this PR.

i am a DWoC and SWoC particpant

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari , in my previous DWoC project doc2pen, whenever I pushed such new minor changes like this, that repo's maintainer asked me to do a squash commit. I know that maintainers can choose to do a **squash commit n merge** themselves.
> Do you have any idea why he always said to do a squash commit to me instead of doing it hemself?
> I never asked him this question coz he might get offended or something😗🤔

Hey, maybe because they want the contributors to learn about it and while squashing and may be because he doesn't want to add extra effort to write a commit msg which merging 

In both ways, if you got to learn something out of it. It's well and good :)

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for your hard work. 
Looks good to go :)

**Comment by ashuvssut** [comment]
> > @Kajol-Kumari , in my previous DWoC project doc2pen, whenever I pushed such new minor changes like this, that repo's maintainer asked me to do a squash commit. I know that maintainers can choose to do a **squash commit n merge** themselves.
> > Do you have any idea why he always said to do a squash commit to me instead of doing it hemself?
> > I never asked him this question coz he might get offended or something😗🤔
> 
> Hey, maybe because they want the contributors to learn about it and while squashing and may be because he doesn't want to add extra effort to write a commit msg which merging
> 
> In both ways, if you got to learn something out of it. It's well and good :)
oh😊

**Pull request merged**